docs/sources/lids/_index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Loki Improvement Documents (LIDs)
+weight: 1400
+---
+
+# Loki Improvement Documents (LIDs)
+
+## Purpose
+
+Loki Improvement Documents (_LIDs_) are proposals for modifying Grafana Loki's feature-set and/or processes. These documents serve to promote engagement between with the community and maintainers _before_ making large changes to Grafana Loki. This ensures that we will only work on features that the maintainers and the community _actually want_, implemented in line with Loki's engineering and scalability considerations.
+
+LIDs are **not** required for:
+
+- bugfixes
+- minor features
+- minor process changes
+
+## Creating a LID
+
+Start by opening a PR against this repository, using [this template](./template.md).
+
+All LIDs require a "sponsor". A sponsor is a Grafana Loki maintainer who is willing to shepherd the improvement proposal through its lifecycle from draft through to completion. A sponsor can be found by starting a thread in our [`#loki-dev` channel on Slack](https://slack.grafana.com), which one or more maintainers will respond to and volunteer. If a LID is generated internally by a Grafana Labs employee who is also a maintainer, no sponsor is required.
```

Comment:
Yeah, that's kinda what I was intending, but re-reading it again I can see how it might be read as "will not need a sponsor". I'll tighten up the language, thanks :+1: